### PR TITLE
Release/1.4

### PR DESCRIPTION
**Pull request by sffc**

### CODE REVIEW

File: provider/datagen/Cargo.toml [hunk 1]
```diff
@@ -104,6 +104,8 @@ crlify = { path = "../../utils/crlify" }
 icu = { path = "../../components/icu" }
 postcard = "1"
 simple_logger = { version = "4.1.0", default-features = false }
+num-bigint = { version = "0.4.4", default-features = false }
+num-rational = { version = "0.4", default-features = false }
```
Comment (sffc) [inline provider/datagen/Cargo.toml:108]: @Manishearth This diff appeared from trying to merge `release/1.4` into `main`. This is the dev-dependencies section, but I also see these deps in the dependencies section above. Please advise.

**Comment by Manishearth** [comment]
Is the goal just to make sure the 1.4 commits are reachable from `main`? If so we should do a merge commit with `--theirs` so that there is no diff on main. I can do that if you'd like.

**Comment by sffc** [comment]
The lines are _added_ in https://github.com/unicode-org/icu4x/commit/40b418ca2a460d1827c9c96fbdea2890c121f42b but they were never added on main AFAICT.

File: provider/datagen/Cargo.toml (hunk 1 above)
Comment (Manishearth) [reply provider/datagen/Cargo.toml:108]: I suggest editing out these two lines and `commit --amend`.

Comment (sffc) [reply provider/datagen/Cargo.toml:108]: Ok so you confirmed that these lines added on release/1.4 should not be merged to main? If so I will proceed with that understanding (`commit --amend` or a fresh PR)

Comment (Manishearth) [reply provider/datagen/Cargo.toml:108]: Yes.

**Review by Manishearth — APPROVED** [review]

**Comment by sffc** [comment]
#4541

**Pull request closed without merging**